datahub-web-react/src/app/entity/shared/types.ts

```diff
@@ -23,11 +23,12 @@ import { FetchedEntity } from '../../lineage/types';
 
 export type EntityTab = {
     name: string;
-    component: React.FunctionComponent;
+    component: React.FunctionComponent<{ properties?: any }>;
```

Comment:
properties should not be `any`. Let's make the type explicit.

Reply:
but different tabs may have different property , isnt it ?

Reply:
added new type to properties